forwarder/parse_commandline_args.py

```diff
@@ -62,32 +62,60 @@ def parse_args():
     parser.add_argument(
         "--config-topic",
         required=True,
-        help="<host[:port][/topic]> Kafka broker/topic to listen for commands",
+        help="<[[SASL_MECHANISM\\]username@]host[:port][/topic]> Kafka broker/topic to listen for commands",
```

Comment:
I had to come up with a connection string format and opted for `SASL_MECHANISM\user@host:port/topic`.

The sasl_mechanism part is not very standard.
Any other ideas on how to do this better?
(I did not use a separate argument because we have 4 different broker arguments, so we would need 4 additional to do it that way)